# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/websockets/remove-own-iframe-during-onerror.window.js.ini
# Commit: 781a851a0e4b
# Full Hash: 781a851a0e4b7d8c34c41a97a06a05934e2d62f2
# Author: Sebastian Hengst <archaeopteryx@coole-files.de>
# Date: 2021-04-10 03:36:58
# Description:
#   Bug 1703998 - set websockets/remove-own-iframe-during-onerror.window.html as intermittently passing on Android WebRender debug. DONTBUILD
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D111374
# ==============================================================================

diff -r 3708fca6d466 -r 781a851a0e4b testing/web-platform/meta/websockets/remove-own-iframe-during-onerror.window.js.ini
--- a/testing/web-platform/meta/websockets/remove-own-iframe-during-onerror.window.js.ini	Fri Apr 09 08:31:52 2021 +0000
+++ b/testing/web-platform/meta/websockets/remove-own-iframe-during-onerror.window.js.ini	Fri Apr 09 08:36:14 2021 +0000
@@ -4,7 +4,7 @@
 
 [remove-own-iframe-during-onerror.window.html?wpt_flags=h2]
   expected:
-    if (os == "android") and debug and webrender: ["ERROR", "TIMEOUT"]
+    if (os == "android") and debug and webrender: ["ERROR", "TIMEOUT", "OK"]
     if (os == "android") and debug and not webrender: ["TIMEOUT", "ERROR"]
     if (os == "android") and not debug and webrender: ["TIMEOUT", "ERROR", "CRASH"]
     if (os == "android") and not debug and not webrender: ["TIMEOUT", "ERROR", "CRASH"]
